core/src/scss/core/fonts/_fonts.scss

```diff
@@ -8,11 +8,13 @@
 //
 // Style guide: Core.Fonts
 
-// Sans font: Source Sans Pro.
-@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap');
+// Sans font: Source Sans 3.
+@warn "Sans serif font weight 200, 300, and 900 will be removed in Decanter Version 7.";
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+3:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap');
 
-// Serif font: Source Serif Pro.
-@import url('https://fonts.googleapis.com/css2?family=Source+Serif+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap');
+// Serif font: Source Serif 4.
+@warn "Serif font weight 200, 300, and 900 will be removed in Decanter Version 7.";
+@import url('https://fonts.googleapis.com/css2?family=Source+Serif+4:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap');
```

Comment:
Update the font imports here. Also added a warning that we're removing 200, 300, and 900 font weight in Decanter 7. Most people don't use 900 in their projects, 200 and 300 are not accessible 🤷🏼‍♀️ If they somehow really need it, they can always add it back into their own projects

Reply:
Cool.

Reply:
@yvonnetangsu We are using the 900 font weight on our ChEM-H subtheme project.